.gitignore

```diff
@@ -4,3 +4,4 @@ test/data/
 test/assets/module/*/config/module.config.php
 clover.xml
 coveralls-upload.json
+docker/data/
```

Comment:
I don't see any usage of this folder, can we remove it?

Reply:
Was a remnant; removed